.gitignore

```
@@ -1,4 +1,5 @@
 build
+build-native
 builddir
```

I think you can unify these;

```suggestion
build*
```